integrated_channels/integrated_channel/tasks.py

```diff
@@ -52,18 +52,16 @@ def transmit_learner_data(username, channel_code, channel_pk):
         channel_pk (str): Primary key for identifying integrated channel
 
     """
+    start = time.time()
     api_user = User.objects.get(username=username)
     integrated_channel = INTEGRATED_CHANNEL_CHOICES[channel_code].objects.get(pk=channel_pk)
     LOGGER.info('Processing learners for integrated channel using configuration: %s', integrated_channel)
     try:
         integrated_channel.transmit_learner_data(api_user)
     except Exception:  # pylint: disable=broad-except
-        exception_message = (
-            'Transmission of learner data failed for user "{username}" and for integrated '
-            'channel with code "{channel_code}" and id "{channel_pk}".'.format(
-                username=username,
-                channel_code=channel_code,
-                channel_pk=channel_pk,
-            )
+        LOGGER.exception(
+            'Transmission of learner data failed for user "%s" and for integrated '
+            'channel with code "%s" and id "%s".', username, channel_code, channel_pk
```

Comment:
Can we standardize on brackets `[]` instead of double-quotes `""` for wrapping log values?

Reply:
:+1: